### PR TITLE
Distress members now get their zoom reset.

### LINKED ISSUE #609: Ghost Zoom Distress Beacon keeps after ERT spawn
## Reproduction:
image

### PR DESCRIPTION
**Pull request by DominikPanic**
Fixes #609

### CODE REVIEW

File: code/datums/emergency_calls/clf.dm [hunk 1]
```diff
@@ -33,6 +33,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/clf.dm]:
```suggestion
		H.client?.change_view(world.view)
```
And remove the `if` check.

File: code/datums/emergency_calls/colonists.dm [hunk 2]
```diff
@@ -21,6 +21,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/colonists.dm]: Same here.

File: code/datums/emergency_calls/deathsquad.dm [hunk 3]
```diff
@@ -29,6 +29,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/deathsquad.dm]: Same.

File: code/datums/emergency_calls/freelancers.dm [hunk 4]
```diff
@@ -31,6 +31,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/freelancers.dm]: Same.

File: code/datums/emergency_calls/pizza.dm [hunk 5]
```diff
@@ -22,6 +22,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/pizza.dm]: Same.

File: code/datums/emergency_calls/pmc.dm [hunk 6]
```diff
@@ -37,6 +37,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/pmc.dm]: Same.

File: code/datums/emergency_calls/upp.dm [hunk 7]
```diff
@@ -36,6 +36,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/upp.dm]: Same.

File: code/datums/emergency_calls/upp_commando.dm [hunk 8]
```diff
@@ -35,6 +35,9 @@
 	if(original)
 		qdel(original)
 
+	if(H.client)
+		(H.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/upp_commando.dm]: Same.

File: code/datums/emergency_calls/xeno.dm [hunk 9]
```diff
@@ -25,10 +25,12 @@
 
 	var/mob/living/carbon/Xenomorph/new_xeno
 
+	if(original.client)
+		(original.client.change_view(world.view))
```
Comment (Rohesie) [inline code/datums/emergency_calls/xeno.dm]: Same.

File: code/datums/emergency_calls/colonists.dm (hunk 2 above)
Comment (DominikPanic) [reply code/datums/emergency_calls/colonists.dm]: Smh not making it easy for me

**Pull request merged**